ui/src/testing/stubs/runtime-api-stub.ts

```diff
@@ -25,6 +26,11 @@ export const defaultGceConfig = (): GceConfig => ({
   machineType: 'n1-standard-4',
 });
 
+export const defaultGceWithPdConfig = (): GceWithPdConfig => ({
```

Comment:
TODO!  Our "default runtime" for testing is GCE without a PD... which is very much not our default anymore

Reply:
yes - in the specific case of Deleted runtimes.

We did try to remove this type in https://github.com/all-of-us/workbench/pull/7930 and that's part of what inspired this deeper dig

Reply:
Deleted runtimes are used to pre-populate configurations for new runtimes.  Maybe we can have something like a DeletedRuntimeConfiguration to represent this, instead of a regular Runtime with status=Deleted and gceConfig=defined

Reply:
Thanks for the explanation, that's helpful to know